### ISSUE TITLE
cannot transfer local files to remote

### ISSUE THREAD

**Issue by fu123456** [stsquad/dired-rsync#15]
I used the command "dired-rsync" to transfer a local file to the remote, however, Emacs outputs errors as follows:
sending incremental file list
rsync: mkdir "/ssh:xxxx@xxxxxxx/home/xxxx/project" failed: No such file or directory (2)
rsync error: error in file IO (code 11) at main.c(664) [Receiver=3.1.3]

"Process completed.
The window will be closed automatically in 5 seconds."
How can I solve this issue? Thanks.

**Comment by stsquad** [comment]
Does the directory /home/xxxx/project or /home/xxxxx exist on the remote end?

**Comment by fu123456** [comment]
Yes, I am very sure that there exist these two directory. Note I use Spacemacs in my Manjaro system.

**Comment by fu123456** [comment]
I use Dired-mode open these two directories together in a window (one directory corresponds a buffer), so they exist.

**Comment by fu123456** [comment]
@stsquad

**Comment by stsquad** [comment]
You've obscured the actual names - any special characters or other such things in the paths?

**Comment by fu123456** [comment]
Hi, I has this problem again. Do you test this package by ssh? rsync in Emacs does not know ‘/ssh:user@******"(does not work), but rsync in Terminal does know 'user@******' (does work).
